feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/thread/MetisThreadUi.kt

```diff
@@ -287,7 +305,9 @@ private fun PostAndRepliesList(
                     verticalArrangement = Arrangement.spacedBy(8.dp)
                 ) {
                     PostWithBottomSheet(
-                        modifier = Modifier.padding(top = 8.dp),
+                        modifier = Modifier
+                            .padding(top = 8.dp)
+                            .testTag(testTagForPost(post.standalonePostId)),
```

Comment:
Notice that this will add the test tag also to the BottomSheet, I am not sure whether this is intended

Reply:
I think it's only added to the PostItem, since the modifier is not passed to PostContextBottomSheet (see PostWithBottomSheet.kt)